src/php/manifest.json

```
@@ -1,5 +1,5 @@
 {
-	"version": "1.0.5",
+	"version": "1.0.6",
```

```suggestion
	"version": "1.1.0",
```

Let's bump minor version, due to https://github.com/devcontainers/images/pull/864 ?